primus.js

```diff
@@ -621,15 +621,29 @@ Primus.prototype.initialise = function initialise(options) {
     }
   }
 
+  primus.addGlobalListeners();
+
+  return primus;
+};
+
+Primus.prototype.addGlobalListeners = function () {
```

Comment:
As you prefer - I was a bit too much trying to separate things, maybe... Having all this "global" listeners logic moved into ```addGlobalListeners``` and corresponding ```removeGlobalListeners```...